src/components/CreateRules/UploadRules.vue

```diff
@@ -235,17 +298,30 @@ export default class UploadRules extends Mixins(DocumentMixin) {
 @import '@/assets/styles/theme.scss';
 header {
   p {
-    padding-top:0.5rem
+    padding-top: 0.5rem
   }
 }
 
 ul {
-  margin-left: 25px !important;
-  padding-left: 25px !important;
+  margin-left: 45px;
+  padding-left: 0px;
+  list-style: none;
+}
+
+// styling used to have custom bullets that use material design icons
+ul li::before {
+  // mdi-circle-small
+  content: '\F09DF';
+  font-family: 'Material Design Icons';
+  margin-left: -24px;
+  padding-left: 0;
+  font-size: 24px;
```

Comment:
As this is currently a "one of" solution, I'd feel more comfortable if v-icons were used explicitly here, for example:
https://github.com/bcgov/business-create-ui/blob/c81dbda78f2d7aa36ce5d2ef07a26fedfef810d2/src/components/AddPeopleAndRoles/PeopleAndRoles.vue#L29

Alternately, this should be implemented globally in a way that can be reused.

Reply:
i was looking at the code you had referenced before implementing this.  i can't remember but there was an issue that stopped me from going with this approach.  will take another look...

Reply:
just tried the implementation using `v-icon` again.  the issue i was having had to do with wrapping. 

![image](https://user-images.githubusercontent.com/6685223/132881359-a5fdfbde-fa90-47ca-8dae-1cfe1a4491d0.png)

Reply:
That can be solved with columns.

Reply:
good call.  will give that a try

Reply:
Or maybe -ve margin on the v-icon only.

Reply:
updated to use `v-icon` solution like `PeopleAndRoles` component.  for scenarios where wrapping was req'd, used columns.  i like this solution better.  my original solution felt a bit hacky.